universal-application-tool-0.0.1/app/services/export/ExporterService.java

```diff
@@ -77,25 +78,14 @@ public String getProgramAllVersionsCsv(long programId) throws ProgramNotFoundExc
     ImmutableList<ProgramDefinition> allProgramVersions =
         programService.getAllProgramDefinitionVersions(programId).stream()
             .collect(ImmutableList.toImmutableList());
-
     CsvExportConfig exportConfig = generateDefaultCsvExportConfig(allProgramVersions);
     CsvExporter csvExporter = exporterFactory.csvExporter(exportConfig);
 
     ImmutableList<Application> applications =
-        allProgramVersions.stream()
-            .map(ProgramDefinition::id)
-            .flatMap(
-                (programVersionId) -> {
-                  try {
-                    return programService
-                        .getSubmittedProgramApplications(programVersionId)
-                        .stream();
-                  } catch (ProgramNotFoundException e) {
-                    throw new RuntimeException(
-                        "Cannot find a program we are trying to generate CSVs for.", e);
-                  }
-                })
-            .collect(ImmutableList.toImmutableList());
+        programService
+            .getSubmittedProgramApplicationsAllVersions(
+                programId, PaginationSpec.MAX_PAGE_SIZE_SPEC, Optional.empty())
```

Comment:
Goog java practices say to doc ambiguous arguments at the call site.  

So

/* searchNamePart */ Optional.empty()

Reply:
done